build/packaging/linux/conf/kcnd.conf

```diff
@@ -22,12 +22,12 @@ TXPOOL_NONEXEC_SLOTS_ACCOUNT=16384
 TXPOOL_LIFE_TIME="5m"
 
 # rpc options setting
-RPC_ENABLE=0 # if this is set, the following options will be used
-RPC_API="klay" # available apis: admin,debug,klay,miner,net,personal,rpc,txpool,web3
-RPC_PORT=8551
-RPC_ADDR="0.0.0.0"
-RPC_CORSDOMAIN="*"
-RPC_VHOSTS="*"
+HTTP_ENABLE=0 # if this is set, the following options will be used
```

Comment:
If this config is updated, then it may cause problems with the old configuration. Don't we need backward compatibility for this?

Reply:
I don't think we need to backward compatibility for config files. Rather we could leave a comment in the file : `This configuration options are for v1.6.4 and upper`.
If you are going to make backward compatibility, this line could be changed to : 
```
RPC_ENABLE=0  # for v1.6.3 and under. Same with HTTP_ENABLE
HTTP_ENABLE=0 # for v1.6.4 and upper. If this is set, the following options will be used
```